extension/src/sidebar.css

```diff
@@ -1,6 +1,48 @@
-.promnesia body {
-    font-size: 15px;
-    color: black;
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;600&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+/* Ligt mode */
+.promnesia {
+    --opacity: 1;
+    --bg: rgba(255, 254, 252, var(--opacity));
+    --bg1: #e8eef0;
+    --bg2: #f3f4f6;
+    --bg3: #ddd;
+    --bg4: #ccc;
+    --color: #222;
+    --color2: #999;
+    --accent-color: 178, 102, 40;
+    --accent-color2: #2aa198;
+}
+
+/* Dark mode */
+/*
+@media (prefers-color-scheme: dark) {
+    .promnesia {
+        --bg: #202124;
+        --bg1: #000;
+        --bg2: #303134;
+        --bg3: #171717;
+        --bg4: #000;
+        --color: #bdc1c6;
+        --color2: #999;
+    }
+}
+*/
+
+.promnesia {
+    --header-bg: var(--bg1);
+    --source-color: rgb(var(--accent-color));
+    --source-bg: var(--bg2);
+    --num-color: var(--bg2);
+    --visits-item-bg: var(--bg2);
+    --locator-color: var(--accent-color2);
+    --date-bg: var(--bg);
+    --date-color: var(--color);
+    --time-color: var(--color2);
+
+    font-family: "PT Serif", serif;
```

Comment:
hmm imo it's subjective -- shall we keep it in your private config?

Reply:
yes